build.gradle

```diff
@@ -123,6 +123,12 @@ subprojects { project ->
   sourceCompatibility = JavaVersion.VERSION_1_8
   targetCompatibility = JavaVersion.VERSION_1_8
 
+  compileJava.options.encoding = 'UTF-8'
```

Comment:
Delete? The second one applies to `compileJava` as it is a `JavaCompile` instance.